systemvm/agent/noVNC/app/ui.js

```diff
@@ -1551,6 +1553,17 @@ const UI = {
         }
     },
 
+    toggleShift() {
+        const btn = document.getElementById('noVNC_toggle_shift_button');
+        if (btn.classList.contains("noVNC_selected")) {
+            UI.sendKey(KeyTable.XK_Shift_L, "ShiftLeft", false);
+            btn.classList.remove("noVNC_selected");
+        } else {
+            UI.sendKey(KeyTable.XK_Control_L, "ShiftLeft", true);
```

Comment:
```suggestion
            UI.sendKey(KeyTable.XK_Shift_L, "ShiftLeft", true);
```

Reply:
Thanks for the heads up @davidjumani! I've just updated this.